### ISSUE TITLE
Antivirus expiry times get screwed up on some web browsers

### ISSUE THREAD

**Issue by redxdev** [redxdev/hvzsite#17]
Not sure why this happens yet. For some reason we need to deal with timezone offsets for antiviruses but not for missions/polls so this doesn't make much sense.

**Comment by weinzach** [comment]
Care to explain the issue? Haven't test run the system on our end yet but could try to assist if we can see the problem.

**Comment by redxdev** [comment]
I actually haven't been able to reproduce this myself but other admins for our game have experienced it a number of times.

Missions/polls both have a date input that as of now is handled correctly across all browsers as far as I'm aware. Antiviruses, however, for some reason need their times adjusted when displaying or when sending back to the server (see [admin-antivirus-create](https://github.com/redxdev/hvzsite/blob/master/client/app/routes/admin-antiviruses-create.js) and [admin-antivirus-edit](https://github.com/redxdev/hvzsite/blob/master/client/app/routes/admin-antiviruses-edit.js)).

What makes this a problem is that some mobile browsers (I haven't confirmed which ones) like to handle their times differently it seems, and the result is that times get messed up if you try to create/edit an antivirus from those browsers.

To test this yourself - create an antivirus and note the exact time you set it to expire. Save it, and check if the time you set is indeed the time that ends up listed on the antivirus list.

**Comment by weinzach** [comment]
Yeah no issues found here. I know mobile browsers can handle time differently but since these are being sent via string from a form to the API I see no reason for them to have offset issues.
